fastFM/tests/test_ffm.py

```diff
@@ -3,7 +3,10 @@
 
 import numpy as np
 import scipy.sparse as sp
-from numpy.testing import assert_almost_equal, assert_equal
+from nose.tools import assert_true
```

Comment:
Is there any advantage over just `assert condition`? That's what I'm using everywhere else.